answer_questions_using_vector_search_weaviate/README.md

```diff
@@ -0,0 +1,28 @@
+# Answer questions by performing similarity search on embedding vectors stored in Weaviate
```

Comment:
```suggestion
# Answer questions by performing a similarity search on embedding vectors stored in Weaviate
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/60/commits/b266f95fb9a512010774915db533652910a42502.